phlop/procs/runtimer.py

```
@@ -26,63 +24,99 @@
-                    check=check,
-                    env=benv,
-                    capture_output=capture_output,
-                    close_fds=True,
-                    **kwargs,
-                    **ekwargs,
+        def go():
+            if popen:
+                self._popen(*kwargs, **ekwargs)
+            else:
+                self._run(
+                    check=check, capture_output=capture_output, *kwargs, **ekwargs
```

Ruff

Star-arg unpacking after a keyword argument is strongly discouraged

(B026)